docs/changes.txt

```diff
@@ -271,6 +272,10 @@ wxGTK:
 - Fix loading WebKit2 extension when using wxWebView (Scott Talbert, #23497).
 - Fix editing cells without values in wxDataViewCtrl (#23523).
 - Fix uninitialized wxKeyEvent::m_isRepeat (jolz, #23593).
+- Also improve size computation of wxTextCtrl (Alex Shvartzkop, #23610).
```

Comment:
"Also improve" should be replaced by just "Improve" because after moving the changelog entries to the correct place, there is no longer any context for "Also".

Reply:
Indeed, thanks for noticing this.